dolweb/downloads/templates/downloads-index.html

```diff
@@ -112,8 +112,20 @@ <h1>{% trans "Stable versions" %}</h2>
 
 <div id="other-linux">
     <h1>{% trans "Linux distributions" %}</h1>
-    <p>{% blocktrans %}Ubuntu users can install a PPA
-    for development and stable versions of Dolphin here: <a href="https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu">Installing Dolphin</a>
+    <p>{% blocktrans %}Ubuntu users on 16.04 and 18.04 can install a PPA
+    for development and stable versions of Dolphin here - users on 20.04 or newer (or Linux Mint, Pop OS, Zorin, etc) must compile or use the Flatpak): <a href="https://wiki.dolphin-emu.org/index.php?title=Installing_Dolphin#Ubuntu">Installing Dolphin</a>
+    {% endblocktrans %}</p>
+    <p>{% blocktrans %}
+    Arch Linux users can download Dolphin via <a href="https://archlinux.org/packages/community/x86_64/dolphin-emu/">pacman</a>
```

Comment:
"Arch Linux, Fedora, and openSUSE can install the `dolphin-emu` package with their package manager. However, distro packages might be slightly out of date. Compile from source for the latest version."

so we don't need to repeat the same paragraph 3 times